src/main/java/com/epam/architecture/RESTws/DTO/BookDTO.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.architecture.RESTws.DTO;
```

Comment:
Packages must be always in lowercase 
```suggestion
package com.epam.architecture.edpoint.dto;
```